### DISCUSSION TITLE
Node: When integration tests are enabled, code Generation Fails with Property DataType list<class> or Class Itself

### DISCUSSION THREAD

**Discussion by Sreenuraj** [radical-sh/public-docs#16] [Issues]
Issue Description:
If integration tests are enabled, When defining a property with a data type of list<class> or a direct reference to a class in the schema, the code generation process fails. It seems to struggle with handling nested classes or lists, resulting in an error during code generation.

Steps to Reproduce:

https://github.com/radical-sh/public-docs/assets/11501324/484c4f7f-775f-41e0-b1ea-ff997b821a70

**Comment by senthilsivanath** [comment]
Resolved.
